### ISSUE TITLE
Exit with returncode 0 (Installer stuck at "Extracting game data")

### ISSUE THREAD

**Issue by staykov** [lutris/lutris#2215]
I am trying to install Jade Empire Special Edition GOG using the installer from here:
https://lutris.net/games/jade-empire-special-edition/

I am running Lutris 0.5.2.1 on Arch Linux.
Installer goes through the whole installation then gets stuck at the end
with this:
```
Done.
Waiting on children
Exit with returncode 0
```
Here's the whole output:
https://gist.github.com/staykov/f913b1d31ab6d9e46a21f0cf2476a135

Any ideas?

**Comment by AccountOneOff** [comment]
Duplicate of #2173
